Fabric.Authorization.API/scripts/AzureAD-Migration.ps1

```diff
@@ -0,0 +1,192 @@
+#Requires -RunAsAdministrator
+#Requires -Version 5.1
+#Requires -Modules PowerShellGet, PackageManagement
+
+param(
+    [PSCredential] $credential,
+    [ValidateScript({
+        if (!(Test-Path $_)) {
+            throw "Path $_ does not exist. Please enter valid path to the install.config."
+        }
+        if (!(Test-Path $_ -PathType Leaf)) {
+            throw "Path $_ is not a file. Please enter a valid path to the install.config."
+        }
+        return $true
+    })] 
+    [string] $installConfigPath = "$PSScriptRoot\install.config",
+    [switch] $quiet
+)
+
+Import-Module -Name .\AzureAD-Utilities.psm1 -Force
+Import-Module -Name .\Install-Authorization-Utilities.psm1 -Force
+Import-Module ActiveDirectory
+
+# Import Fabric Install Utilities
+$fabricInstallUtilities = ".\Fabric-Install-Utilities.psm1"
```

Comment:
Same potential CD issue - this one is potentially bypassed since you'll download it, but this may end up with multiple copies of the utilities if this script is executed from multiple locations.